GameBoard/Controllers/GameEventParticipationController.cs

```diff
@@ -48,6 +53,35 @@ public async Task<IActionResult> ExitGameEvent(ExitGameEventViewModel exitGameEv
             return RedirectToAction("Index", "Home");
         }
 
+        [HttpGet]
+        public async Task<IActionResult> RemoveFromGameEvent(int gameEventId, string userName)
+        {
+            var gameEvent = await _gameEventService.GetGameEventAsync(gameEventId);
+
+            if (gameEvent == null)
+            {
+                return Error.FromController(this).Error("Error!", "Game event you're trying to remove doesn't exist.", HttpStatusCode.NotFound);
```

Comment:
This message is misleading, we're not trying to remove a Game Event here, but a user from a Game Event.

Reply:
Fixed.